src/Controls/tests/Core.UnitTests/Layouts/FlexLayoutTests.cs

```diff
@@ -133,6 +133,64 @@ public void UnconstrainedWidthChildrenHaveWidth()
 			Assert.Equal(100, flexFrame.Width);
 		}
 
+		[Fact]
+		public void FlexLayoutPaddingShouldBeAppliedCorrectly_RowDirection()
+		{
+			// Arrange
+			var padding = 16;
+			var root = new Grid();
+			var flexLayout = new FlexLayout { Padding = padding };
+			var view1 = new TestLabel();
+			var view2 = new TestLabel();
+
+			root.Add(flexLayout);
+			flexLayout.Add(view1 as IView);
+			flexLayout.Add(view2 as IView);
+
+			// Act
+			var measure = flexLayout.CrossPlatformMeasure(1000, 1000);
+			flexLayout.CrossPlatformArrange(new Rect(Point.Zero, measure));
+
+			var view1Frame = flexLayout.Children[0].Frame;
+			var view2Frame = flexLayout.Children[1].Frame;
+
+			var leftPadding = view1Frame.X;
+			var topPadding = view1Frame.Y;
+			var rightPadding = measure.Width - view2Frame.Right;
+
+			// Assert
+			Assert.Equal(padding, leftPadding);
+			Assert.Equal(padding, rightPadding);
+			Assert.Equal(padding, topPadding);
+			Assert.Equal(view1.Width, view1Frame.Width);
```

Comment:
Can this check use a more specific value? 
I think this comparison isn't comparing anything interesting

Check for the specific values you expect the width of view1 and view2 to be